tests/assets/seedImage.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: elemental.cattle.io/v1beta1
+kind: SeedImage
+metadata:
+  name: seed-image-%POOL_TYPE%-%CLUSTER_NAME%
+  namespace: fleet-default
```

Comment:
In the assets file it's not specified (check another file for an example), it's done in the code itself by specifying the NS. The NS is defined as a variable in the workflow, to allow a more dynamic configuration.